src/components/HeroBackground/index.js

```diff
@@ -29,7 +29,7 @@ const HeroBackground = ({ children, ...props }) => (
     }
     {...props}
   >
-    {children}
+    <div css={content}>{children}</div>
```

Comment:
I think `z-index: 2` css should be added in component `<ContentBox />` instead of adding another layer here. What do you think?

Reply:
yes, you are right. 
this new layer will also include the `<Lottie />` thats not good.
i will change this.